internal/api/router.go

```diff
@@ -0,0 +1,12 @@
+package api
+
+import (
+	"github.com/gorilla/mux"
+	"net/http"
+)
+
+func NewRouter() *mux.Router {
+	r := mux.NewRouter()
+	r.HandleFunc("/health", HealthCheckHandler).Methods(http.MethodGet)
```

Comment:
Would lead to this being lower case as well.
```suggestion
	r.HandleFunc("/health", healthCheckHandler).Methods(http.MethodGet)
```